src/mmw/js/src/modeling/views.js

```diff
@@ -324,6 +328,52 @@ var ScenarioButtonsView = Marionette.ItemView.extend({
             compareUrl: compareUrl
         };
     },
+
+    showCompare: function() {
+        var scenarios = this.projectModel.get('scenarios'),
+            runoffTable = [
+                {
+                    name: "Runoff",
+                    unit: "cm",
+                    values: scenarios.map(function(s) {
+                        return s.get('results')
+                                .findWhere({ name: "runoff" })
+                                .get('result')
+                                .runoff.modified.runoff;
```

Comment:
This is extremely brittle, and is meant to be replaced. I put this stuff in `modeling/views` initially since it seemed like there was enough context to direct the Compare Window, but as the amount of configuration to be supplied has grown, I think we should pass minimal things to the Compare Window and abstract away all this decision making stuff there in. I'll add a note that explains this, but wouldn't want to spend more time on this right now. Not sure which of the cards already made would be a good fit for this task. We may need to make a new one, in which case I will.